kokoro/build_binaries_rust.sh

```diff
@@ -38,6 +38,7 @@ readonly generated_binaries=(
     ./enclave_apps/target/x86_64-unknown-none/release/oak_echo_raw_enclave_app
     ./enclave_apps/target/x86_64-unknown-none/release/oak_functions_enclave_app
     ./enclave_apps/target/x86_64-unknown-none/release/oak_functions_insecure_enclave_app
+    ./enclave_apps/target/x86_64-unknown-none/release/oak_orchestrator
```

Comment:
Double check the binary name `oak_orchestrator`: what if a second orchestrator variant comes up, then the mainline is taken. Maybe OK, may be not.